src/common/runtime-bind.js

```diff
@@ -0,0 +1,23 @@
+define([
+	"./runtime-key"
+], function( runtimeKey ) {
+
+return function( args, cldr, fn, runtimeArgs ) {
+	var fnName = /^function\s+([\w\$]+)\s*\(/.exec( fn.toString() )[ 1 ],
+		locale = cldr.locale;
+
+	fn.runtimeKey = runtimeKey( fnName, locale, args );
+
+	// Stringify arguments.
+	args = JSON.stringify( args ).slice( 1, -1 );
```

Comment:
Is `args` guaranteed to never contain a cyclical structure? Afaik that's the only reason for `JSON.stringify` to throw a `TypeError`